static/app/views/explore/components/table.tsx

```diff
@@ -74,10 +76,19 @@ export function useTableStyles({
 }
 
 export const TableBody = GridBody;
-export const TableBodyCell = GridBodyCell;
+export const TableRow = GridRow;
+export const TableBodyCell = styled(GridBodyCell)`
+  &:first-child {
+    > * {
+      text-align: left;
+    }
+  }
+`;
+
 export const TableHead = GridHead;
-export const TableHeadCell = GridHeadCell;
 export const TableHeader = Header;
 export const TableHeaderActions = HeaderButtonContainer;
 export const TableHeaderTitle = HeaderTitle;
-export const TableRow = GridRow;
+export const TableHeadCell = styled(GridHeadCell)<{align?: Alignments}>`
+  justify-content: ${({isFirst, align}) => (isFirst ? 'left' : align ? align : 'right')};
```

Comment:
Why is the first one aligned to the left and the rest based on `align`?

Reply:
This just makes sure that the header aligns with body, as addressed in the comment above.